multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+/*.yaml
+*~
```

Comment:
What specifically are you trying to ignore here? Can we have explicit files instead of * expansion?